Description: relax dependency on env_logger
 Debian has only a newer release of env_logger,
 which apparently works just fine.
 .
 This patch relaxes the dependency from 0.8.0
 to 0.10.
Author: Jonas Smedegaard <dr@jones.dk>
Author: Peter Michael Green <plugwash@debian.org> (update for quickcheck 1.0.3)
Author: Matthias Geiger <werdahias@riseup.net> (update for env-logger update)
Last-Update: 2023-08-28
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -28,7 +28,7 @@
 [lib]
 name = "quickcheck"
 [dependencies.env_logger]
-version = "0.8.2"
+version = "0.10"
 optional = true
 default-features = false
 
